### PR TITLE
Add page 2 grid layout

### PR DESCRIPTION
**Pull request by shad674**
Description:
- grid layout of page 2 of the website

Notes:
- Ralph review it and comment if you find something interesting

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -8,7 +8,21 @@
...
+    <!-- Grid Layout -->
+    <div class="grid-layout">
+      <div class="pet-layout">
+        <h2>Pets</h2>
+      </div>
+      <div class="nature-layout">
+        <div class="transparent-background"></div>
+        <h2>Nature</h2>
+      </div>
+      <div class="friend-layout"><h2>Friends</h2></div>
```
Comment (shad674) [inline index.html:20]: I'll fix the styling here later, it doesn't clean it every time I save it

**Review by rgogs — COMMENTED** [review]
good job

**Review by rgogs — COMMENTED** [review]
i like it

**Review by rgogs — COMMENTED** [review]
good code

**Review by rgogs — COMMENTED** [review]
thanks for deleting the heic files

**Review by rgogs — APPROVED** [review]

**Pull request merged**